tests/unit/conftest.py

```diff
@@ -156,6 +158,44 @@ def fixture_harness_server_name_configured(harness: Harness) -> Harness:
     harness.update_config({"server_name": TEST_SERVER_NAME})
     harness.enable_hooks()
     harness.begin_with_initial_hooks()
+    container: ops.Container = harness.model.unit.get_container(SYNAPSE_CONTAINER_NAME)
+    container.push(f"{SYNAPSE_CONFIG_PATH}", f'server_name: "{TEST_SERVER_NAME}"', make_dirs=True)
```

Comment:
I think the fstring on the first path can be removed :D
```suggestion
    container.push(SYNAPSE_CONFIG_PATH, f'server_name: "{TEST_SERVER_NAME}"', make_dirs=True)
```

Reply:
Oops...Fixed ;)